sagemaker-inference-recommender/xgboost/xgboost-inference-recommender.ipynb

```diff
@@ -0,0 +1,793 @@
+{
```

Comment:
<p>Add a Conclusion section summarizing what the user accomplished and learned, and any recommended next steps. If there are any resources to be deleted, add a Cleanup section with the deletions, but that may not apply to this notebook.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3073/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='50'/>

Reply:
<p>Done</p>